joomla/joomla-cms
# [Associations] Adding direct link to language filter plugin when there are no associations

### Summary of Changes
Previously when there were no associations and one wanted to use the Multilingual Associations extension, there was a message to enable the `Language Filter` plugin with no direct link, one had to go to Modules-> Search for it and publish it. This patch saves you a click.


### Testing Instructions
Install Joomla! without extra languages and multilingual, go to Multilingual Associations extension, check if you get the correct message and if the link is correct.


### Before
![image](https://cloud.githubusercontent.com/assets/16468676/25859640/11d62788-34d7-11e7-93fd-9eae3d32a937.png)



### After patch
![image](https://cloud.githubusercontent.com/assets/16468676/25859663/2187c038-34d7-11e7-86c3-c79d9757a882.png)
